cmd/admin-user-add.go

```diff
@@ -166,7 +171,27 @@ func fetchUserKeys(args cli.Args) (string, string) {
 		secretKey = args.Get(2)
 	}
 
-	return accessKey, secretKey
+	if argCount < 4 {
```

Comment:
I would not require to enter these parameters if they are not provided as arguments and just populated them as nil.